homeassistant/components/bmw_connected_drive/lock.py

```diff
@@ -84,17 +83,17 @@ async def async_unlock(self, **kwargs: Any) -> None:
     def _handle_coordinator_update(self) -> None:
         """Handle updated data from the coordinator."""
         _LOGGER.debug("Updating lock data of %s", self.vehicle.name)
+        # Set default attributes
+        self._attr_extra_state_attributes = self._attrs
```

Comment:
Not related to this PR, but why does the `bmw_connected_drive` integration add the car name and id as state attributes, why is it not enough that the same information is present in the device info?

Reply:
When we introduced the device info to the integration, we were not allowed to store the VIN (car id) as `sw_version`, see https://github.com/home-assistant/core/pull/39585#discussion_r545421706.
I could try opening a PR and storing it to `hw_version`, however this probably would get the same answer?

I'm with you that this should be cleaned up, we also do not need the `_attr_attribution` tbh.

Reply:
HW version is also not a good fit. Maybe we should add `serial_number` to the device object?

Reply:
I should be able to add it to `core`. However I have never worked with Typescript/on the frontend so would need help there.